docs/docsite/rst/playbook_guide/playbooks_loops.rst

```diff
@@ -256,6 +256,41 @@ When you run a task with ``until`` and register the result as a variable, the re
 
 If ``until`` is not specified, the task will retry until the task succeeds but at most ``retries`` times.
 
+.. _retry_with_timeout:
+
+Retrying a task with timeout
+----------------------------
+
+You can use the ``timeout`` keyword to set a time limit for the execution of the task code. 
```

Comment:
This seems too generic to me here. Like it's inserting details about a keyword that isn't necessarily about loops.